src/clients/dotnet/docs.zig

```diff
@@ -38,12 +38,7 @@ pub const DotnetDocs = Docs{
 
     .examples = "",
 
-    .client_object_documentation =
-    \\The `Client` class is thread-safe and for better performance, a
-    \\single instance should be shared between multiple concurrent
-    \\tasks. Multiple clients can be instantiated in case of connecting
-    \\to more than one TigerBeetle cluster.
-    ,
+    .client_object_documentation = "",
```

Comment:
@batiati I actually removed this section for the dotnet client since it was already covered in the paragraph just before.